plugins/CoreHome/lang/en.json

```diff
@@ -122,6 +122,7 @@
         "DevicesSubcategoryHelp": "The Devices section helps you understand the technology that your visitors are using to access your site. You will see reports on the type of device and specific models to enable you to optimise your site for the most popular devices.",
         "SoftwareSubcategoryHelp": "The Software section shows the operating systems, browsers and plugins that your visitors are using to access the site so that you can optimise your site to ensure it is fully compatible with the most popular configurations..",
         "EngagementSubcategoryHelp1": "The Engagement section provides reports that help to quantify how many new and returning visitors you get. You can also review reports that break down the average time and number of pages per visit, as well as the number of times a visitor has been to your site and the most common number of days between visits.",
-        "EngagementSubcategoryHelp2": "This can help you to optimise for frequency and high-interaction visits in addition to maximising your reach."
+        "EngagementSubcategoryHelp2": "This can help you to optimise for frequency and high-interaction visits in addition to maximising your reach.",
+        "PeriodHasOnlyRawData": "It looks like reports for this period have not been processed yet. Do you want to see what's happening now? Check out the %1$sVisits log%2$s or choose a different date period until the reports are generated."
```

Comment:
optional feedback: when we refer to existing menu names, it may be a bit easier to write simply:

> Check out the %s or choose

and then replace the placeholder by the link and the menu name (using translation key so there is no duplicate of menu names.)

Reply:
good catch, sure, I will do it 👍

Reply:
While it might be ok to do it in this case, we should not try to avoid duplication in translations too much. Often it will be better to have duplication as it is easier for the translator to know the context. Also in some languages names might be written in another way based on their usage and including the version that was translated without any context might actually be grammatically incorrect sometimes...

Reply:
Thinking about it again, we even should change it back in this case. For example in German `the` can be translated with `der`, `die` or `das` depending on the word that follows after it. As a translator doesn't know what follows in this case, it actually can't be translated correctly.

Reply:
+1 for that. Avoiding duplication we stopped doing in most cases many years ago as it leads to poor translations.

Reply:
interesting, makes sense.. could we document this best practise in this relevant section maybe https://developer.matomo.org/guides/translations#reusing-translations ?

Reply:
@mattab that's about a different "reusing" regarding using translations from core basically. Created https://github.com/matomo-org/developer-documentation/pull/463 . We didn't document it earlier since someone would usually not do this anyway but also good to specifically mention to not do it